### PR TITLE
feat: Add WarHornFeature

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

File: common/src/main/java/com/wynntils/models/territories/GuildAttackTimerModel.java [hunk 1]
```diff
@@ -116,7 +118,19 @@ void processChanges(ScoreboardSegment segment) {
...
+
+        // Signal events for the new vars
+        for (TerritoryAttackTimer attackTimer : attackTimers) {
+            if (attackTimers.stream()
+                    .anyMatch(territoryAttackTimer ->
+                            territoryAttackTimer.territory().equals(attackTimer.territory()))) {
+                continue;
+            }
+
+            WynntilsMod.postEvent(new GuildWarQueuedEvent(attackTimer));
```
Comment (ShadowCat117) [inline common/src/main/java/com/wynntils/models/territories/GuildAttackTimerModel.java]: Shouldn't this be comparing against oldTimers?

File: common/src/main/java/com/wynntils/features/embellishments/WarHornFeature.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@ConfigCategory(Category.EMBELLISHMENTS)
+public class WarHornFeature extends Feature {
+    private static final ResourceLocation WAR_HORN_ID = new ResourceLocation("wynntils:war.horn");
+    private static final SoundEvent WAR_HORN_SOUND = SoundEvent.createVariableRangeEvent(WAR_HORN_ID);
+
+    @SubscribeEvent
+    public void onWarQueued(GuildWarQueuedEvent event) {
+        McUtils.playSoundAmbient(WAR_HORN_SOUND);
```
Comment (ShadowCat117) [inline common/src/main/java/com/wynntils/features/embellishments/WarHornFeature.java]: Worth adding the volume/pitch configs to this?

**Comment by ShadowCat117** [comment]
You also need to add the `war.horn` sound to sounds.json

File: common/src/main/java/com/wynntils/features/embellishments/WarHornFeature.java (hunk 2 above)
Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/embellishments/WarHornFeature.java]: sure

File: common/src/main/java/com/wynntils/models/territories/GuildAttackTimerModel.java (hunk 1 above)
Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/models/territories/GuildAttackTimerModel.java]: Yeah, I couldn't test it when I wrote it..

File: common/src/main/java/com/wynntils/models/territories/GuildAttackTimerModel.java [hunk 3]
```diff
@@ -116,7 +118,22 @@ void processChanges(ScoreboardSegment segment) {
             }
         }
 
+        // FIXME: This model should use chat messages to parse new wars,
+        //        and only fall back to scoreboard on first join.
```
Comment (kristofbolyai) [inline common/src/main/java/com/wynntils/models/territories/GuildAttackTimerModel.java:122]: To be fixed with `AttackTimerOverlayFeature (Improved handling for territories cut off scoreboard, either vertically, or horizontally)`

**Review by ShadowCat117 — APPROVED** [review]

**Pull request merged**